src/address.rs

```diff
@@ -202,40 +202,6 @@ impl<A, Rc: RefCounter> Address<A, Rc> {
         BroadcastFuture::new(msg, self.0.clone())
     }
 
-    /// Attaches a stream to this actor such that all messages produced by it are forwarded to the
-    /// actor. This could, for instance, be used to forward messages from a socket to the actor
-    /// (after the messages have been appropriately `map`ped). This is a convenience method over
-    /// explicitly forwarding a stream to this address and checking when to stop forwarding.
-    ///
-    /// Often, this should be spawned onto an executor to run in the background. **Do not await this
-    /// inside of an actor** - this will cause it to await forever and never receive any messages.
-    ///
-    /// **Note:** if this stream's continuation should prevent the actor from being dropped, this
-    /// method should be called on [`Address`]. Otherwise, it should be called
-    /// on [`WeakAddress`].
-    pub async fn attach_stream<S, M, K>(self, stream: S)
-    where
-        K: Into<KeepRunning> + Send,
```

Comment:
Could we get rid of KeepRunning now?

Reply:
Yes.

Reply:
Can it be done in this PR? `attach_stream` was the only reason it was kept this long